electron/windows.mjs

```
@@ -0,0 +1,95 @@
+  }
+};
+
+export function createMainWindow() {
+  const displays = screen.getAllDisplays();
+  // Find the primary display or the one with the highest resolution
+  const primaryDisplay = displays.find((display) => display.isPrimary);
+  const display =
+    primaryDisplay ||
+    displays.reduce((a, b) =>
+      a.workArea.width * a.workArea.height >
+        b.workArea.width * b.workArea.height
```

ESLint

[error] Delete `··`

(prettier/prettier)